docs/docs/supply-chain/sbom.md

```diff
@@ -17,6 +17,10 @@ $ trivy image --format spdx-json --output result.json alpine:3.15
 $ trivy fs --format cyclonedx --output result.json /app/myproject
 ```
 
+```
+$ trivy k8s cluster --format cyclonedx --output result.json
```

Comment:
this is the kbom creation referenced in the Kubernetes scanning docs? It is a bit unclear what the difference here is between kbom and sbom

Reply:
KBOM is BOM for the cluster (what you k8s is made of). SBOM is usually BOM for the image (what your application is made of)

Reply:
Maybe providing a sentence or two of clarification e.g. "SBOMs for Kubernetes i.e. KBOM
A KBOM specifies XXX" @chen-keinan what do you think?